### PR TITLE
AN-37 Anki flashcard export

### PR DESCRIPTION
**Pull request by SimonEndres**
Können wir den mergen @lkno0705 ?

### CODE REVIEW

File: valid_data.pt
Comment (lkno0705) [inline valid_data.pt:1]: This should not be here

File: train_data.pt
Comment (lkno0705) [inline train_data.pt:1]: This should also not be here

File: txt_models 2.ipynb
Comment (lkno0705) [inline txt_models 2.ipynb:1]: This should also not be here.. The T5 model has nothing to do with the flashcard analysis and export.. This should be on a seperate branch in a seperate Pull Request with a second backlog item linked to it...

File: notebooks/dataexploration/data_exploration.ipynb
Comment (lkno0705) [inline notebooks/dataexploration/data_exploration.ipynb:1]: For now this is ok, but we'll need to add some plots to this later, for the report

**Review by Niklas-23 — CHANGES_REQUESTED** [review]
No, this can definitely not be merged that way. Several things have gone wrong here:

1. We always use English on GitHub, please.
2. This PR contains commits from @pranav1305  and @SimonEndres to different tasks . WHY WEREN'T DIFFERENT BRANCHES USED?
3. For each PR a jira ticket is created and the PR title contains the corresponding jira tag.

To solve this problem, one needs to copy out all T5 content from @pranav1305  and then delete it to create a clean PR. Then create a new branch and upload the T5 content there.

**Review by lkno0705 — CHANGES_REQUESTED** [review]
We can't merge it like this.. This PR is kind of a mess.
1. The corresponding link to the Backlogitem on Jira is missing
2. Why is the T5 model here? This should not be here. This should be a seperate Backlogitem with a corresponding branch.
4. The T5 model isn't finished yet
5. The train and validation pt files, shouldn't be versioned on git at all, as large files that are in a binary non human readable format will cause unnecessary problems
6. The T5 model notebook isn't saved in the chorrect folder structure either.

So, In terms of the data export and exploration part, this PR is fine.. but because of the rest, we can't merge it like that. Please remove the T5 stuff from this PR and Branch and move it to a second one, with a corresponding backlog item. Also the correct Backlogitem for this PR should be added. I'd suggest to create a new branch from main and copy the T5 stuff over and then deleting it from this branch with a new commit.

Also @pranav1305 why are these commits verified? Did you upload them using the web interface??

**Review by lkno0705 — APPROVED** [review]
lgtm

**Review by Niklas-23 — APPROVED** [review]
Now it looks good

**Pull request merged**